src/main/java/bio/terra/service/snapshot/SnapshotService.java

```diff
@@ -167,8 +173,24 @@ public SnapshotSummaryModel retrieveSnapshotSummary(UUID id) {
      * @return a SnapshotModel = API output-friendly representation of the Snapshot
      */
     public SnapshotModel retrieveAvailableSnapshotModel(UUID id) {
+        return retrieveAvailableSnapshotModel(id, getDefaultIncludes());
+    }
+
+    /**
+     * Convenience wrapper around fetching an existing Snapshot object and converting it to a Model object.
+     * Unlike the Snapshot object, the Model object includes a reference to the associated cloud project.
+     *
+     * Note that this method will only return a snapshot if it is NOT exclusively locked.
+     * It is intended for user-facing calls (e.g. from RepositoryApiController), not internal calls that may require
+     * an exclusively locked snapshot to be returned (e.g. snapshot deletion).
+     * @param id in UUID formant
```

Comment:
typo, also seems redundant if the type of the argument is `UUID` (not `String`)
```suggestion
     * @param id in UUID format
```

Reply:
Ah shoot, I missed this when doing cut and paste.  I'll fix in the other cases too